pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -387,6 +410,17 @@ func (r *ReconcileClusterDeployment) reconcile(request reconcile.Request, cd *hi
 
 		cdLog.Debug("cluster is already installed, no processing of provision needed")
 		r.cleanupInstallLogPVC(cd, cdLog)
+
+		if cd.Spec.ClusterMetadata != nil &&
+			cd.Spec.ClusterMetadata.AdminKubeconfigSecretRef.Name != "" &&
+			(cd.Status.WebConsoleURL == "" || cd.Status.APIURL == "") {
```

Comment:
This is not something that I expect us to do for this PR, but we should get in the habit of enforcing that status fields are set to the correct value on each reconcile instead of just simply set to something. So, here in particular, it would be better for the controller to determine the web console URL on every reconcile and check that the value set in the `WebConsoleURL` matches the correct value.